hedera-node/docs/design/services/smart-contract-service/cancun-fork-support.md

```diff
@@ -0,0 +1,131 @@
+# Cancun Fork Support
+
+## Purpose
+
+The Ethereum "Cancun" hardfork introduces a number of changes to the EVM that will need to be implemented to maintain EVM
+Equivalence. There are four different HIPs covering this feature (tracked in  
+  epic [#11697](https://github.com/hashgraph/hedera-services/issues/11697):
+* [HIP-865](https://hips.hedera.com/hip/hip-865): Add EVM Support for transient storage and memory 
+  copy Cancun opcodes (issue [#11699](https://github.com/hashgraph/hedera-services/issues/11699))
+* [HIP-866](https://hips.hedera.com/hip/hip-866): Add EVM compatibility for non-supported Cancun blob features
+  (issue [#11700](https://github.com/hashgraph/hedera-services/issues/11700))
+* [HIP-867](https://hips.hedera.com/hip/hip-867): Add Smart Contract Services Support for KZG Point 
+  Evaluation Precompiled Function (issue [#11701](https://github.com/hashgraph/hedera-services/issues/11701))
+* [HIP-868](https://hips.hedera.com/hip/hip-868): Support Cancun Self-Destruct  Semantics in Smart
+  Contract Services (issue [#11702](https://github.com/hashgraph/hedera-services/issues/11702))
+
+This document describes how the HIPs will be implemented in Hedera Services.
+
+Generally speaking there are four strategies that will be adopted:
+
+1. Adopt without change. This includes the `MCOPY`, `TSTORE`, and `TLOAD` operations as well as the KZG precompile.
+2. Adapt with Hedera Considerations. This includes the `SELFDESTRUCT` changes.
+3. Do not adopt but stub out placeholders. This includes the `VERSIONEDHASH` and `BLOBBASEFEE` operations.
+4. Do not adopt. This includes all Blob handling and the "Type 3" blob transaction.
+
+### Example User Story
+
+<!-- **TODO(Nana): more specific user stories highlighting the capabilities to users** -->
+
+* As a smart contract developer, I want to use current versions of solidity that may generate opcodes that are only
+  available in the Cancun fork.
+* As a Hedera developer, I want to preserve maximum future design space to adopt, adopt, or not adopt blobs.
```

Comment:
adopt wording repetition?

Reply:
There's a lot of adoption going on in that sentence for sure.

Reply:
Addressed,